vm-setup/roles/common/defaults/main.yml

```
@@ -79,4 +106,11 @@ external_network:
           addr: "{% if external_network_cidr_v4|ipv4 != False %}127.0.0.1{% else %}::1{% endif %}"
       srvs: "{{dns_externalsrvs | default([])}}"
 
-networks: "{{ provisioning_network + external_network }}"
+# Provisioning network is bridged and external network is nated
+networks_mixed: "{{ provisioning_network_bridge + external_network }}"
+# Both networks are nated
+networks_nated: "{{ provisioning_network_nat + external_network }}"
+# Enable only nated networks
+networks_nat_only: "false"
```

Why not set it from env here directly?
```suggestion
networks_nat_only: "{{ lookup('env', 'ENABLE_NATED_PROVISIONING_NETWORK') | bool }}"
```
After that we could set `networks` to the correct value also so we don't have to do all the `set_fact` tasks

---

You are right, I will change it.

---

Although that will introduce an additional export in the bash common.sh script or in one of the numbered bash scripts, but I guess as we define most of the defaults for env vars (if not all) in those scripts it is fine.